### PR TITLE
Update dependencies; kill hacks that are no longer needed

### PR DESCRIPTION
**Pull request by Kovensky**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,11 +30,11 @@
     "react-visjs-timeline": "^1.5.0",
     "redux": "^4",
     "redux-thunk": "^2.2.0",
-    "semantic-ui-css": "^2.3.1",
+    "semantic-ui-css": "^2.4.0",
     "semantic-ui-react": "^0.82.3",
     "stable": "^0.1.8",
     "toposort": "^2",
-    "url-search-params-polyfill": "^4.0.1",
+    "url-search-params-polyfill": "^5.0.0",
```
Comment (Kovensky) [inline package.json]: This is not actually a breaking change, just was accidentally published as one 🤷‍♀️

File: package.json [hunk 2]
```diff
@@ -78,32 +78,7 @@
...
-    "@babel/plugin-transform-runtime": "^7.0.0",
-    "@babel/preset-env": "^7.0.0",
-    "@babel/preset-flow": "^7.0.0",
-    "@babel/preset-react": "^7.0.0",
-    "@babel/runtime": "^7.0.0",
-    "@babel/template": "^7.0.0",
-    "@babel/traverse": "^7.0.0",
-    "@babel/types": "^7.0.0",
-    "babel-preset-react-app": "file:./preset-react-app-custom"
+    "babel-eslint": "^9.0.0"
```
Comment (Kovensky) [inline package.json]: For some reason one of the `postcss` dependencies is depending on an older `babel-eslint` (???)

Probably a `devDependency` mistakenly published as `dependency`

**Comment by ackwell** [comment]
Cancelled 'cus was SSH'd in. Looks like that `@babel/core` resolution has fixed CI's `EPIPE`, gonna merge and see how she flies.

File: package.json [hunk 3]
```diff
@@ -78,32 +78,8 @@
     "format": "minimal"
   },
   "resolutions": {
-    "babel-core": "7.0.0-bridge.0",
```
Comment (Kovensky) [inline package.json]: oops :abloblamp:

**Pull request merged**